### PR TITLE
ddcutil: init at 0.8.2

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
ddccontrol doesn't seem to work with my GPU.

###### Things done

- [x] Tested using sandboxing
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change (none)
- [x] Tested execution of all binary files
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/misc/ddcutil/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = [
+    i2c-tools udev libgudev
+    glib libusb libdrm xorg.libXrandr
+  ];
+
+  meta = with stdenv.lib; {
+    homepage    = http://www.ddcutil.com/;
+    description = "Query and change Linux monitor settings using DDC/CI and USB";
+   license     = licenses.gpl2;
```
Comment (rasendubi) [inline pkgs/tools/misc/ddcutil/default.nix]: Alignment is a bit off.

**Review by rasendubi — APPROVED** [review]
Looks good to me.

When you say it doesn't work with your GPU, would you like a little help with testing or package is good to go as is?

**Comment by rnhmjoj** [comment]
ddccontrol used to run mostly (because it generated huge amounts of warnings about not properly supported hardware) fine when I was using the Intel integrated GPU. After switching to an AMD GPU it gives an error such `DDC/CI at /dev/i2c-* is unusable`.

**Comment by rnhmjoj** [comment]
Ah, we misunderstood each other. In the motivation I meant that I packaged this tool (ddcutil) because what I was previously using [ddccontrol](https://github.com/NixOS/nixpkgs/blob/56904d7c423f2b13b37fbd29f39bbb4b52bc7824/pkgs/tools/misc/ddccontrol/default.nix) started giving problems. This package is good and the software is working perfectly.

**Pull request merged**